gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -958,11 +958,11 @@ gst_tensor_filter_compare_tensors (GstTensorsInfo * info1,
     }
 
     line =
-        g_strdup_printf ("%2d : %s | %s %s\n", i, left, right,
+        g_strdup_printf ("%2d : %s | %s %s\n", i, GST_STR_NULL(left), GST_STR_NULL(right),
         g_str_equal (left, right) ? "" : "FAILED");
```

Comment:
This might incur failure in indentation check: http://nnsuite.mooo.com/nnstreamer/ci/repo-workers/pr-checker/1410-201904301508290.043931007385254-27fdd70d8a83517ce3baf0ce1171e3d6320fedaf/report/indent-format.patch

Reply:
Thanks! I fixed it.